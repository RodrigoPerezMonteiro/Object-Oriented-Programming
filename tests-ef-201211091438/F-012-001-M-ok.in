6
3
Porco palha
carne de porco
30
batata palha
70
1
0
7
3
All Pizza
2
Porco palha
1000
3
3
Porco palha
0
0
0
