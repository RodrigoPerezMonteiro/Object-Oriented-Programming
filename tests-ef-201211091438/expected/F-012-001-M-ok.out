Gestão de uma Cadeia de Restaurantes
1 - Novo
2 - Abrir
3 - Guardar
4 - Guardar Como...
5 - Gestão de Clientes
6 - Gestão de Alimentos
7 - Gestão de Restaurantes
8 - Consultas
0 - Sair
Escolha uma opção: Gestão de Alimentos
1 - Visualizar alimentos
2 - Registar alimento simples
3 - Registar alimento preparado
4 - Alterar alimento simples
5 - Descrever alimento preparado
0 - Sair
Escolha uma opção: Nome a dar ao alimento: Identificador do alimento (nome): Percentagem do alimento (inteiro entre 0 e 100): Identificador do alimento (nome): Percentagem do alimento (inteiro entre 0 e 100): Gestão de Alimentos
1 - Visualizar alimentos
2 - Registar alimento simples
3 - Registar alimento preparado
4 - Alterar alimento simples
5 - Descrever alimento preparado
0 - Sair
Escolha uma opção: VEGETABLE|alface|1
VEGETABLE|azeite|9
VEGETABLE|batata|4
VEG|batata palha
MEAT|carne de porco|14
FISH|linguado|6
NOVEG|linguado grelhado
NOVEG|Porco palha
VEG|salada mista
VEGETABLE|tomate|2
Gestão de Alimentos
1 - Visualizar alimentos
2 - Registar alimento simples
3 - Registar alimento preparado
4 - Alterar alimento simples
5 - Descrever alimento preparado
0 - Sair
Escolha uma opção: Gestão de uma Cadeia de Restaurantes
1 - Novo
2 - Abrir
3 - Guardar
4 - Guardar Como...
5 - Gestão de Clientes
6 - Gestão de Alimentos
7 - Gestão de Restaurantes
8 - Consultas
0 - Sair
Escolha uma opção: Gestão de Restaurantes
1 - Visualizar restaurantes
2 - Registar restaurante
3 - Menu de um restaurante
0 - Sair
Escolha uma opção: Identificador do restaurante (nome): Gestão de um Restaurante
1 - Visualizar pratos do dia
2 - Criar prato do dia
3 - Calcular calorias
4 - Disponibilizar prato do dia
5 - Indisponibilizar prato do dia
6 - Alterar desconto
0 - Sair
Escolha uma opção: Identificador do alimento (nome): Quantidade do alimento (inteiro): Preço do prato do dia (inteiro): Gestão de um Restaurante
1 - Visualizar pratos do dia
2 - Criar prato do dia
3 - Calcular calorias
4 - Disponibilizar prato do dia
5 - Indisponibilizar prato do dia
6 - Alterar desconto
0 - Sair
Escolha uma opção: Identificador do prato do dia (nome): Porco palha|7770
Gestão de um Restaurante
1 - Visualizar pratos do dia
2 - Criar prato do dia
3 - Calcular calorias
4 - Disponibilizar prato do dia
5 - Indisponibilizar prato do dia
6 - Alterar desconto
0 - Sair
Escolha uma opção: Gestão de Restaurantes
1 - Visualizar restaurantes
2 - Registar restaurante
3 - Menu de um restaurante
0 - Sair
Escolha uma opção: Gestão de uma Cadeia de Restaurantes
1 - Novo
2 - Abrir
3 - Guardar
4 - Guardar Como...
5 - Gestão de Clientes
6 - Gestão de Alimentos
7 - Gestão de Restaurantes
8 - Consultas
0 - Sair
Escolha uma opção: 
